docs/REST API/rest-api.md

```diff
@@ -7178,45 +7306,130 @@ Deletes the session specified by ID. When the user associated with the session n
         "hosts_count": 1
       },
       {
-        "id": 2,
+        "id": 32,
         "name": "1Password – Password Manager",
-        "version": "2.3.7",
+        "versions": "2.10.0",
         "source": "chrome_extensions",
-        "extension_id": "aeblfdkhhhdcdjpifhhbdiojplfjncoa",
         "browser": "chrome",
-        "generated_cpe": "cpe:2.3:a:1password:1password:2.3.7:*:*:*:*:chrome:*:*",
-        "vulnerabilities": null,
-        "hosts_count": 2
+        "extension_id": "aeblfdkhhhdcdjpifhhbdiojplfjncoa",
+        "generated_cpe": "cpe:2.3:a:1password:1password:2.19.0:*:*:*:*:chrome:*:*",
+        "hosts_count": 345,
+        "vulnerabilities": null
       }
-    ]
+    ],
+    "meta": {
+      "has_next_results": false,
+      "has_previous_results": false
+    }
 }
 ```
 
-### Count software
+### Get software title
```

Comment:
@marko-lisica similar comment as here.

Maybe we rename this header to simply "Get software"?